packages/flutter_tools/lib/src/ios/iproxy.dart

```diff
@@ -51,6 +59,8 @@ class IProxy {
         '$hostPort:$devicePort',
         '--udid',
         deviceId,
+        if (_logger.isVerbose)
+          '--debug',
```

Comment:
This isn't related to the name change, but I want to pull it over from https://github.com/flutter/flutter/pull/60408 as well.